apps/frontend/src/app/5_pages/BitocracyPage/contexts/ProposalContext.types.ts

```diff
@@ -38,6 +45,14 @@ export type ProposalCreationDetails = {
   text: string;
 };
 
+export type ProposalCreationTreasuryDetails = {
+  treasuryType: ProposalTreasuryType;
+  treasuryTypeContract: string;
+  recipientAddress: string;
+  amount: string;
+  token: SupportedTokens;
+};
```

Comment:
Put additional info to the `ProposalCreationParamter` type, for example as `extra` prop or similar. Also verify with @tiltom as he already extending it too.